coredump/criu_coredump/coredump.py

```diff
@@ -558,9 +558,8 @@ class mmaped_file_info:
             fields.append(("start" + str(i), ctypes.c_long))
             fields.append(("end" + str(i), ctypes.c_long))
             fields.append(("file_ofs" + str(i), ctypes.c_long))
-        for i in range(len(infos)):
-            fields.append(
-                ("name" + str(i), ctypes.c_char * (len(infos[i].name) + 1)))
+        for i,info in enumerate(infos):
```

Comment:
```suggestion
        for i, info in enumerate(infos):
```
please add a space after comma